pkg/convert/arb.go

```diff
@@ -135,7 +135,7 @@ func ToArb(translation model.Translation) ([]byte, error) {
 	dst["@@locale"] = translation.Language
 
 	for _, msg := range translation.Messages {
-		dst[msg.ID] = removeEnclosingBrackets(msg.Message)
+		dst[msg.ID] = removeEscapeSpecialChars(removeEnclosingBrackets(msg.Message))
```

Comment:
Would parsing the MF2 and using the text node work better? This way we get the real value (unserialized).